gdal/doc/source/api/gdalogrin_python.rst

```diff
@@ -0,0 +1,229 @@
+.. _gdalogrin_python:
+
+================================================================================
+GDAL/OGR in Python
+================================================================================
+
+This Python package and extensions are a number of tools for programming and manipulating the `GDAL <http://www.gdal.org/>`__ Geospatial Data Abstraction Library. Actually, it is two libraries -- GDAL for manipulating geospatial raster data and 

```

Comment:
paragraph should be rewritten as just "This Python package and extensions are a number of tools for programming and manipulating GDAL"

Reply:
Fixed